src/main/java/com/textmessenger/model/entity/dto/CommentToFront.java

```diff
@@ -0,0 +1,33 @@
+package com.textmessenger.model.entity.dto;
+
+import com.textmessenger.model.entity.Comment;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class CommentToFront {
+  private long id;
+  private String content;
+  private long user;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "user" private field. ([squid:S1068](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_avAJjEPbGfqJY3&open=AWWU5_avAJjEPbGfqJY3)

[ISSUE_KEY]: AWWU5_avAJjEPbGfqJY3